app/config.py

```diff
@@ -101,43 +137,36 @@ def get_abs_path(file_path: str):
 # Useful when calling Postfix from an external network
 POSTFIX_SUBMISSION_TLS = "POSTFIX_SUBMISSION_TLS" in os.environ
 
-if "OTHER_ALIAS_DOMAINS" in os.environ:
-    OTHER_ALIAS_DOMAINS = eval(
-        os.environ["OTHER_ALIAS_DOMAINS"]
-    )  # ["domain1.com", "domain2.com"]
-else:
-    OTHER_ALIAS_DOMAINS = []
+OTHER_ALIAS_DOMAINS = sl_get_env(
+    "OTHER_ALIAS_DOMAINS", default_factory=list
+)  # ["domain1.com", "domain2.com"]
 
 # List of domains user can use to create alias
 if "ALIAS_DOMAINS" in os.environ:
-    ALIAS_DOMAINS = eval(os.environ["ALIAS_DOMAINS"])  # ["domain1.com", "domain2.com"]
+    ALIAS_DOMAINS = os.environ.get("ALIAS_DOMAINS", default="").split(
+        ","
+    )  # ["domain1.com", "domain2.com"]
 else:
     ALIAS_DOMAINS = OTHER_ALIAS_DOMAINS + [EMAIL_DOMAIN]
 
 ALIAS_DOMAINS = [d.lower().strip() for d in ALIAS_DOMAINS]
 
-if "PREMIUM_ALIAS_DOMAINS" in os.environ:
-    PREMIUM_ALIAS_DOMAINS = eval(
-        os.environ["PREMIUM_ALIAS_DOMAINS"]
-    )  # ["domain1.com", "domain2.com"]
-else:
-    PREMIUM_ALIAS_DOMAINS = []
+PREMIUM_ALIAS_DOMAINS: List[str] = sl_get_env(
+    "PREMIUM_ALIAS_DOMAINS", default_factory=list
+)  # ["domain1.com", "domain2.com"]
 
-PREMIUM_ALIAS_DOMAINS = [d.lower().strip() for d in PREMIUM_ALIAS_DOMAINS]
+PREMIUM_ALIAS_DOMAINS = [d.lower().strip() for d in PREMIUM_ALIAS_DOMAINS if d]
 
 # the alias domain used when creating the first alias for user
 FIRST_ALIAS_DOMAIN = os.environ.get("FIRST_ALIAS_DOMAIN") or EMAIL_DOMAIN
 
 # list of (priority, email server)
-EMAIL_SERVERS_WITH_PRIORITY = eval(
-    os.environ["EMAIL_SERVERS_WITH_PRIORITY"]
+EMAIL_SERVERS_WITH_PRIORITY: List[Tuple[int, str]] = sl_get_env(
+    "EMAIL_SERVERS_WITH_PRIORITY", default_factory=list
 )  # [(10, "email.hostname.")]
 
 # these emails are ignored when computing stats
-if os.environ.get("IGNORED_EMAILS"):
-    IGNORED_EMAILS = eval(os.environ.get("IGNORED_EMAILS"))
-else:
-    IGNORED_EMAILS = []
+IGNORED_EMAILS = os.environ.get("IGNORED_EMAILS", default="").split(",")
```

Comment:
should we use `sl_get_env` here :)?

Reply:
in fact ,`email1, email2` is not a expression in Python :(

Reply:
Ah this option is usually put as 

```
IGNORED_EMAILS = ["string1", "string2"]
```

and that's why `eval` was used :)